app/models/user.rb

```diff
@@ -8,6 +8,7 @@ class User < ApplicationRecord
 
   has_many :posts, dependent: :destroy
   has_many :communities, dependent: :destroy
+  has_many :comments, dependent: :destroy
```

Comment:
Kruta, ka pieliki tos dependent destroy, bet es tev varu sarežģīt situāciju :D

Kruta fīča ir, ja lietotājs izdzēš savu profilu, viņa komentāri paliek sistēmā bet pie user rāda nezināms lietotājas vai tml.

Bet to varbūt piereģistrē nākotnē sev kā uzdevumu izstrādāt.

Reply:
Nja, es aju apdomaju ka posts varetu nedzest kad lietotajs deleto sevi, varetu same ar commentiem izdarit.